torch/nn/modules/tests/test_activation.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
```

Comment:
Could you move this next to the other tests in `test/` please?

Reply:
See @cpuhrsch 's comment below. I'm moving it to /test/test_native_mha.py since this change enables a new run mode for native mha.